experimental/DoubleAndHyperComplexes/src/Morphisms/simplified_complexes.jl

```diff
@@ -228,7 +228,7 @@ function (fac::SimplifiedChainFactory)(d::AbsHyperComplex, Ind::Tuple)
       w = Tinv[i]
       new_entries = Vector{Tuple{Int, elem_type(base_ring(w))}}()
       for (real_j, b) in w
-        j = findfirst(==(real_j), J)
+        j = findfirst(k->k==real_j, J)
```

Comment:
```suggestion
        j = findfirst(==(real_j), J)
```
why was this changed back (compared to https://github.com/oscar-system/Oscar.jl/pull/4126)?